eskipfile/watch_test.go

```diff
@@ -49,19 +51,42 @@ func createFileWith(content string, t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer os.Remove(tmpfile.Name())
 
 	_, err = tmpfile.WriteString(content)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	err = os.Rename(tmpfile.Name(), testWatchFile)
+	err = moveFile(tmpfile.Name(), testWatchFile)
```

Comment:
why not pass the tmpFile, which is already open?

Reply:
Yeah I copy pasted this function as the first thing I did to get tests passing, I'll do as you suggest and see if it can be cleaned up a bit before taking the pr out of draft